adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +106,62 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    if (adConfiguration.childDirectedTreatment) {
+        [GADMediationAdapterPangle setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    }
+    _bannerRenderer = [[GADPangleRTBBannerRenderer alloc] init];
+    [_bannerRenderer renderBannerForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadInterstitialForAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration
                          completionHandler:(GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
-}
-
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+    if (adConfiguration.childDirectedTreatment) {
+        [GADMediationAdapterPangle setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    }
+    _interstitialRenderer = [[GADPangleRTBInterstitialRenderer alloc] init];
+    [_interstitialRenderer renderInterstitialForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
                        completionHandler:(GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+    if (adConfiguration.childDirectedTreatment) {
```

Comment:
If `adConfiguration.childDirectedTreatment` is `nil`, it means the publisher hasn't specified child directed treatment, which means we need to pass `-1` to Pangle SDK.

Reply:
update this invoke way `[GADMediationAdapterPangle setCoppa:(adConfiguration.childDirectedTreatment ? adConfiguration.childDirectedTreatment.integerValue : -1)];`